package.json

```diff
@@ -3,18 +3,19 @@
   "version": "1.0.0",
   "type": "module",
   "dependencies": {
+    "concurrently": "^6.2.0",
```

Comment:
seems like these changes just reordered the dependencies. no change to versions. should be good here.